includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -216,13 +218,22 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
 		]
 	];
 
-	private $wikiId;
+	private $product;
+	private $productInstanceId;
 	private $lang;
 
-	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+	/**
+	 * DesignSystemGlobalFooterModel constructor.
+	 *
+	 * @param string $product Name of product, ex: fandoms, wikis
+	 * @param int $productInstanceId Identifier for given product, ex: wiki id
+	 * @param string $lang
+	 */
+	public function __construct( $product, $productInstanceId, $lang = self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)